tests/CitizenProject/CitizenProjectManagerTest.php

```diff
@@ -101,10 +100,11 @@ protected function setUp()
 
     protected function tearDown()
     {
-        $this->loadFixtures([]);
+        $this->cleanupContainer($this->container);
```

Comment:
could you call of `ControllerTestTrait::kill()` here ?

Reply:
done